api_specification/api_specification.json

```diff
@@ -0,0 +1,91 @@
+{
+  "swagger" : "2.0",
+  "info" : {
+    "description" : "The api starts with `/v1/<contract-category>/`\neach category will have it's interface.\n- The only required parameters for `DIGITS` are `shortcode` + `currency`.\n- We have explicitly decided not to support back pricing in this endpoint.\n- A future endpoint will have an extra date_pricing parameter, to support back pricing.\n",
```

Comment:
Niggle:
* This should be `its` not `it's`

Background in case you didn't know, because I didn't for *years*:

We probably all know these two things:
* `it's` is short for `it is`
* Normally possessive has an apostrophe: Wei Sheng's drug habit

However:
* Because `it's` is already a contraction, there's an exception for `its` - Things belonging to it; No apostophe

[Better written here](https://www.merriam-webster.com/words-at-play/when-to-use-its-vs-its)

Reply:
I think this description contains what should go into the README.
Aim to keep this really simple, just what the endpoint is for, and nothing else.